lib/assembly-objectfile.rb

```diff
@@ -14,6 +14,14 @@ module Assembly
   #  by the file command, then a check will be made to see if exif data exists...if so, the mimetype returned by the exif data will be used
   #  if no exif data exists, then the mimetype returned by the unix file command will be used
   TRUSTED_MIMETYPES = ['text/plain', 'plain/text', 'application/pdf', 'text/html', 'application/xml'].freeze
+
+  # this is a manual override mapping of file extension to mimetype; if a file with the given extension is found, the mapped
+  #  mimetype will be returned and no further methods will be used - this is used to force a specific mimetype to be returned for
+  #  for a given file extension regardless of what exif or the unix file system command returns
+  #  the mapping format is "extension with period: returned mimetype",  e.g. for any .json file, you will always get `application/json`
+  OVERRIDE_MIMETYPES = {
+    '.json': 'application/json'
```

Comment:
should this be for `.jsonld` also?  @andrewjbtw

Reply:
ahh, good call, that would be wise

Reply:
The mime-type for jsonld would be `application/ld+json` though.

Reply:
I'm hesitant to go down the road of adding more definitions without knowing the expected use yet. The `application/json` case is following a known access system need, and it may be best to come back and add on an as-needed basis.

Reply:
ok, i'm going to go ahead, merge, cut a new release and then bump in common-accessioning, which uses this to add mimetypes to content metadata